protocol-configuration/backupOracleConfig.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from 'hardhat';
+import { BigNumberish } from 'ethers';
+
+const toBN = ethers.BigNumber.from;
+const BNe18 = (x: any) => ethers.constants.WeiPerEther.mul(toBN(x));
+
+export const daiUsdcBackupOracleConfig: BackupOracleConfig = {
+  twapPeriod: 600, // 10 minutes
+  minPoolLiquidity: 50e6, // 50m
+  uniswapPool: '0x5777d92f208679DB4b9778590Fa3CAB3aC9e2168',
+  precision: BNe18(1)
+};
+
+export const ethUsdcBackupOracleConfig: BackupOracleConfig = {
+  twapPeriod: 600, // 10 minutes
+  minPoolLiquidity: 50e6, // 50m
```

Comment:
Please make sure to run these params by Storm and or Delia for all backup oracle parameters.

Reply:
I drafted these values with Storm's input + they're what he's been running the data gathering scripts on. But will confirm before deploy